pom.xml

```diff
@@ -108,14 +108,20 @@
 
         <!-- MicroProfile specs  -->
         <config-version>3.1</config-version>
-        <ft-version>4.1-RC2</ft-version>
+        <ft-version>4.1</ft-version>
         <health-version>4.0.1</health-version>
         <metrics-version>5.1.0</metrics-version> 
         <jwt-version>2.1</jwt-version>
-        <openapi-version>4.0-RC3</openapi-version> 
-        <rest-client-version>4.0-RC2</rest-client-version>
-        <telemetry-version>2.0-RC2</telemetry-version> 
+        <openapi-version>4.0</openapi-version> 
+        <rest-client-version>4.0</rest-client-version>
+        <telemetry-version>2.0</telemetry-version> 
 
+        <!-- other plugin versions-->
+
+        <version.plugin.javadoc>3.7.0</version.plugin.javadoc>
+        <version.plugin.gpg>3.2.4</version.plugin.gpg>
+        <version.plugin.release>3.0.1</version.plugin.release>
+        <version.plugin.staging>1.7.0</version.plugin.staging>
```

Comment:
These versions match those in the microprofile-parent pom.xml.

I'm unsure why we don't use the parent pom in this project.

Reply:
There was a concern with circular dependency in the past. Can revisit next time.